src/languages/es.ts

```diff
@@ -2317,6 +2311,49 @@ export default {
                     },
                 },
             },
+            advancedConfig: {
+                autoSyncDescription: 'Expensify will automatically sync with NetSuite every day.',
+                reimbursedReportsDescription: 'Any time a report is paid using Expensify ACH, the corresponding bill payment will be created in the NetSuite account below.',
+                reimbursementsAccount: 'Reimbursements account',
```

Comment:
Sent you a link to the doc to use for the rest for now: https://expensify.slack.com/archives/C06ML6X0W9L/p1719968288304099